pkg/miner/miner.go

```diff
@@ -63,12 +74,21 @@ func (m *Miner) Stop() {
 }
 
 // Resume resumes the work of all worker functions, each in a separate
-// goroutine.
+// goroutine. If Stop() has been called multiple times, Resume() needs to be
+// called the same number of times. Resume() blocks until the work can be
+// resumed.
 func (m *Miner) Resume() {
```

Comment:
I think `Resume` is not safe for redundant calls. If we have a working miner and call `Resume` by mistake, the latch's counter will become negative and cause a panic. Shouldn't we make redundant `Resume` calls a no-op? It would be also good to document this behavior in the docstring and cover it with a test case.

Reply:
See [this](https://github.com/keep-network/keep-core/pull/3174#discussion_r948823835) comment please.